web/src/copy_messages.ts

```diff
@@ -245,7 +214,22 @@ export function copy_handler(): boolean {
 
```

Comment:
I don't understand why the above `return false` block is not adjusted by this PR.